Tests/Queries/FilterInterpreterTests.cs

```diff
@@ -125,6 +151,34 @@ public void WorksOnStrings()
             Assert.Equal(expected, result);
         }
 
+        [Fact(DisplayName = "Applies filtering on strings")]
+        public void WorksOnStringsWithSpaces()
+        {
+            var companies = Get.Companies(100).ToList().AsQueryable();
+            var expected = companies.Where(c => c.Name == "Awesome Inc.").ToList();
+
+            var query = GetQuery(new[] { "Name" }, new[] { "Aweseom Inc." });
+
+            var result = Query.ApplyFiltering(companies, new FilterContext(query), new CompanyResource())
+                as IQueryable<Company>;
+
+            Assert.Equal(expected, result);
+        }
+
+        [Fact(DisplayName = "Applies filtering on strings with multiple values")]
+        public void WorksOnStringsMultiple()
+        {
+            var people = Get.People(100).ToList().AsQueryable();
+            var expected = people.Where(c => c.LastName == "Russel" || c.LastName == "Comma,Test").ToList();
+
+            var query = GetQuery(new[] { "LastName" }, new[] { "Russel,\"Comma,Test\"" });
```

Comment:
Maybe we can also see what happens on some edge cases/malformed strings such as `"a"b` (no comma), `"ab,c` (no end quote), etc.

FWIW i'm not sure what *should* happen in this case.. I would be fine with a 400 although technically that would be a breaking change.

Reply:
Right now this is failing silently as anything not captured will be ignored. It's certainly possible to add a 400. I am not sure if this would be the best way tho.

Reply:
I would vote that either

1. We throw an error for this (400 bad request)
2. We identify the filter as "malformed" and interpret it as a single filter value

I don't think we should let parts drop, throw 500, etc.

I think you're right that option 2 is probably better.

Reply:
I added a few checks and one more unit test for this.